components/rich_text_editor/rich_text_editor_default.story.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <dt-rich-text-editor
+    v-model="inputValue"
+    :editable="editable"
+    :input-aria-label="inputAriaLabel"
+    :output-format="outputFormat"
```

Comment:
I tried to do the same thing for `autofocus` but it wouldn't work for one reason or another. Like the storybook would even strip this param from the URL when set, so I must be missing something. I'll have to look into it later unless someone knows off the bat what the issue is. At least the linter didn't like the `autofocus` name since it assumes it's the native one, so might need to change that to `auto-focus` or something.